wrappers/python/zxing.cpp

```diff
@@ -47,6 +51,30 @@ Result decode(const Image& image, BarcodeFormat format, bool fastMode, bool tryR
 	return decode(image, {format}, fastMode, tryRotate, hybridBinarizer);
 }
 
+Image encode(std::string text, int width, int height, BarcodeFormat format, int margin, int eccLevel) {
+	MultiFormatWriter writer(format);
+	if (margin >= 0)
+		writer.setMargin(margin);
+	if (eccLevel >= 0)
+		writer.setEccLevel(eccLevel);
+
+	auto bitmap = writer.encode(TextUtfEncoding::FromUtf8(text), width, height).toByteMatrix();
```

Comment:
If you are going to do a pixel-by-pixel initialization of your memory below, then there is no need call toByteMatrix() here effectively doing that twice.